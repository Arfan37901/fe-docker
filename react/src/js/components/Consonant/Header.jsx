import React from 'react';

const Header = () => (
    <header data-testid="consonant-header" className="consonant-header" />
);

export default Header;
