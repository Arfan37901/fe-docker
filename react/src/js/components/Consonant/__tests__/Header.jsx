import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';

describe('Consonant/Header', () => {
    describe('Check snapshots', () => {
        test('should renders correctly', () => {
            const tree = renderer
                .create(<Header />)
                .toJSON();

            expect(tree).toMatchSnapshot();
        });
    });
});
