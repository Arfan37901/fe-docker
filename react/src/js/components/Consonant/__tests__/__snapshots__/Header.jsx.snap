// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Consonant/Header Check snapshots should renders correctly 1`] = `
<header
  className="consonant-header"
  data-testid="consonant-header"
/>
`;
